### DISCUSSION TITLE
Поле селект с края экрана

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#439] [Неопределенное]
Скажите пожалуйста, можно ли поправить ширину контекстного меню, если поле селект крайнее справа? Т.е. чтобы оно не было таким узким.
Заранее спасибо

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/58977262-cab6-408d-9749-25d1006991bf)

**Comment by ttm-alexeypolunin** [comment]
Сделайте поле пишире.

**Reply by mihail-cornev** [reply]
в этом случае всё равно нужно крутить таблицу
